docs/common/new_config_files.asciidoc

```diff
@@ -0,0 +1,31 @@
+After packages upgrades, you should take care to review any changes to configuration files and merge them if required.
+
+To find out which configuration files have changed run following command:
+
+.RHEL-based systems
+[source,bash]
+----
+find /usr/local/pf -name \*.rpmnew
+----
+
+.Debian-based systems
+[source,bash]
+----
+find /usr/local/pf -name "*.dpkg-dist"
+----
+
+The list of files returned are the new versions shipped with PacketFence.
```

Comment:
Compare them to your existing version -> Compare them with your installed versions